code/modules/surgery/surgery.dm

```diff
@@ -64,6 +64,13 @@
 		if(type in SP.advanced_surgeries)
 			return TRUE
 
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		var/obj/item/organ/cyberimp/brain/linkedsurgery/IMP = C.getorganslot(ORGAN_SLOT_SURGICAL_IMPLANT)
+		if(!IMP || (replaced_by in IMP.advanced_surgeries))
+			return .
```

Comment:
```suggestion
			return
```

Reply:
It is not self return anymore.
By the way, I think it should be dot because other lines have that too and it lets you know that it has something to change at return since it has `. = FALSE` or `. = TRUE` there.